docker-compose.yml

```diff
@@ -17,4 +17,6 @@ services:
     environment:
       - SPEC_OPTS
       - LOG_LEVEL # devutils (>= 2.0.4) reads the ENV and sets LS logging
+      - CI # CI=true in Travis-CI
```

Comment:
Are these always specified in the target plugin's configuration or should have some default values?

Reply:
good question, we'll be checking `ENV['CI'] == 'true'` (in specs) so it kind of forces us to think of having `${CI:-false}`
than on the other hand there's no default for the other vars and they just end up empty if not set e.g. `LOG_LEVEL=""`
... kind of inclined on keeping the same convention for all (empty if unspecified), wdyt?

Reply:
I'm fine with it, if there isn't any default value, keep it black. I asked this because on some `docker-compose.yml`(in some plugins, I saw the default value set.
LGTM